src/register/RegistrationPage.jsx

```diff
@@ -744,7 +605,41 @@ class RegistrationPage extends React.Component {
                     helpText={intl.formatMessage(messages['helptext.password'])}
                   />
                 )}
-                { this.addExtraRequiredFields() }
+                <AuthnValidationFormGroup
+                  label={intl.formatMessage(messages['registration.country.label'])}
+                  for="country"
+                  name="country"
+                  type="select"
+                  key="country"
+                  invalid={this.state.errors.country !== ''}
+                  invalidMessage={intl.formatMessage(messages['country.validation.message'])}
+                  className="mb-0"
+                  value={this.state.country}
+                  onClick={(e) => this.handleOnClick(e)}
+                  onBlur={(e) => this.handleOnBlur(e)}
+                  onChange={(e) => this.handleOnChange(e)}
+                  selectOptions={this.getOptions().countryOptions}
+                />
+                <input id="honorCode" name="honorCode" type="hidden" value={this.state.honorCode} required onChange={e => this.handleOnChange(e)} />
```

Comment:
honor code should always be true and i don't think we are allowing user to change its state. The honor code paragraph is meant to be an implicit agreement by user

Reply:
I will remove this and keep it as simple text. Removing this will also fix your concern mentioned above